include/odp/api/spec/std_types.h

```diff
@@ -40,6 +41,24 @@ extern "C" {
  */
 typedef uint32_t odp_percent_t;
 
+/**
+ * @typedef odp_u128_t
+ * 128-bit unsigned integer
+ */
+typedef struct ODP_ALIGNED(16) odp_u128_s {
+	union {
```

Comment:
Doxygen warns that this union needs documentation. You can add a dummy documentation like "128 bit in various word sizes".

You can run doxygen locally with:
```
make clean
make doxygen-doc
```

Reply:
Done.